emissions.md

```diff
@@ -50,15 +50,17 @@ GCAM can be considered as a process model for CO<sub>2</sub> emissions and reduc
 
 CO<sub>2</sub> emissions from limestone used in cement production are also estimated. Limestone consumption has one global emissions factor, however, each region's IO coefficient (limestone / cement) is calibrated to return CDIAC estimates ([CDIAC 2017](#cdiac2017)). (CO<sub>2</sub> from fuel consumed in producing limestone is estimated in the same manner as other fuel consumption.)
 
-CO<sub>2</sub> emissions from gas flaring are not currently included in GCAM.
+### <a name="co2-luc-emissions"/> Fugitive CO<sub>2</sub> Emissions From Fossil Fuel Production (Oil, Gas, and Coal)
+
+Fugitive CO<sub>2</sub> emissions from fossil resource production are also included in GCAM. These include CO<sub>2</sub> emissions resulting from natural gas flaring that occurs at the point of extraction (e.g., oil well flares) as well as CO<sub>2</sub> gas released from oil, gas, or coal resources during the process of extraction. Fugitive CO<sub>2</sub> emissions from fossil resource production in GCAM are initialized from the [CEDS inventory](https://github.com/JGCRI/CEDS) ([Hoesly et al 2018](#Hoesly2018)) and are modeled using the  [Non-CO2 emissions](emissions.html#non-co2-overview) approach.
```

Comment:
Thank you for these suggestions-- I added a link to the IPCC guidelines and included an additional bullet point under the "initialization" section for non-co2 emissions to describe the unconventional oil approach.

Reply:
Thanks Taryn!  Two more small comments:
- For the IPCC reference, can you please put the full citation in the References section at the bottom of the page, and then update the in-text link to link to that citation (rather than direct link to the URL)?  This is consistent with the format for other citations.
- I just noticed that this PR has the wrong target branch (gh-pages is the current live documentation, but we collect all documentation updates for the next release in a separate branch that goes live with the release (this is currently gcam-v6.1).  Once we resolve this last issue, we should either retarget this PR (if possible) or simply remove it and open a new one to the correct target branch.

Reply:
Thanks, Matthew. I've added the full citation with in-text link. I will now close this PR and open a new one to gcam-v6.1.